src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SourcelinkTests.cs

```diff
@@ -32,7 +32,13 @@ public void VerifySourcelinks()
         }
         Directory.CreateDirectory(SourcelinkRoot);
 
-        IList<string> failedFiles = ValidateSymbols(ExtractSymbolsPackages(GetAllSymbolsPackages()), InitializeSourcelinkTool());
+        string symbolsRoot = Directory.CreateDirectory(Path.Combine(SourcelinkRoot, "symbols")).FullName;
```

Comment:
This directory should probably get cleaned up at the end of the test.

Reply:
Parent directory gets deleted when we run smoke-tests the next time. But, I can see how this can delete some unnecessary test artifacts, to keep lower disk usage.

Will fix.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17477/commits/701e39a0b5be1f265f0fc1f4edf87d44d00da78a